test/test_module_tracker.py

```diff
@@ -3,7 +3,9 @@
 from copy import copy
 
 import torch
+from torch import nn
```

Comment:
Nitpicking: this is a nice BE change, but could have been done as part of separate PR :)